scripts/abi_artifact_generator/templates/types.h.jinja_template

```diff
@@ -0,0 +1,42 @@
+#pragma once
+/*( _ )*/
+
+/*#-
+This template file is distributed under the MIT License. See LICENSE.md for details.
+#*/
+
+/*(- generation_notice )*/
+
+#ifdef __SIZEOF_INT128__
+#define INT128_IS_SCALAR 1
+#else
+#define INT128_IS_SCALAR 0
+#endif
+
+#define IS_SCALAR(TYPE, VALUE) \
```

Comment:
What's definition of *scalar* here?
Some `struct`s seem to be classified as scalars.
Add a comment.

Reply:
Right, my bad!